base-cards/date/embedded.js

```diff
@@ -0,0 +1,9 @@
+import { setComponentTemplate } from "@ember/component";
+import { precompileTemplate } from "@ember/template-compilation";
+import templateOnlyComponent from "@ember/component/template-only";
+export default setComponentTemplate(
+  precompileTemplate("Date: {{@model}}", {
+    strictMode: true,
+  }),
+  templateOnlyComponent()
+);
```

Comment:
This is just an aside, but: pzuraq (who made the ember-template-imports addon and the underlying glimmer features it uses) mentioned yesterday wanted to offer a shorthand for all of this. It seems like a good idea. It's a lot of verbosity.